tests/gnrc_tcp/tests/test_01-conn_lifecycle_as_client.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
```

Comment:
On naming: I'd prefer keeping the current style and name the scripts e.g. `01-conn_lifecycle_as_client.py` without the `test_` prefix.

Reply:
Done